mne/io/tests/test_meas_info.py

```diff
@@ -32,6 +33,41 @@ def test_make_info():
     coil_types = set([ch['coil_type'] for ch in info['chs']])
     assert_true(FIFF.FIFFV_COIL_EEG in coil_types)
 
+    assert_raises(TypeError, create_info, 'Test Ch', 1000, None)
+    assert_raises(ValueError, create_info, ['Test Ch'], -1000, None)
+    assert_raises(ValueError, create_info, ['Test Ch'],  1000, ['eeg', 'eeg'])
+    assert_raises(TypeError, create_info, [np.array([1])], 1000, None)
+    assert_raises(TypeError, create_info, ['Test Ch'], 1000, np.array([1]))
+    assert_raises(KeyError, create_info, ['Test Ch'], 1000, 'awesome')
+    assert_raises(TypeError, create_info, ['Test Ch'], 1000, None,
+                  np.array([1]))
```

Comment:
Can you please pass by keyword, it will make it easier to read the tests and pick up their logic.